### DISCUSSION TITLE
v59.0.0 will not run on my M2Pro Mac Mini Sonoma

### DISCUSSION THREAD

**Discussion by Stubedo** [forcedotcom/dataloader#895] [General]
Exports appear to start, but never do. Updated JDK, reinstalled multiple times. v58.0.4 works fine.

**Comment by ashitsalesforce** [comment]
@Stubedo , do you have the console output capture (screenshot or copy/paste) when you ran data loader v59.0.0?

**Comment by ashitsalesforce** [comment]
Closing discussion assuming that the issue is resolved.

**Comment by ashitsalesforce** [comment]
Hi @Shaun2453, Yes, that line seems to be the key. Please take the
following steps:

- Create a new issue so that we can track it as a separate one.
- Provide the SOQL you are trying to execute.
- Mapping file (.sdl extension), if using CLI mode.

Ashit
